kratos/utilities/variable_utils.h

```diff
@@ -800,24 +800,29 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
     {
         KRATOS_TRY
 
-        if(rNodes.size() != 0) {
-            // First we do a check
-            CheckVariableExists(rVar, rNodes);
+        if (rNodes.size() != 0) {
+            // checking the first node to avoid error being thrown in parallel region
+            KRATOS_ERROR_IF_NOT(rNodes.begin()->HasDofFor(rVar)) << "Trying to fix/free dof of variable " << rVar.Name() << " but this dof does not exist in node #" << rNodes.begin()->Id() << "!" << std::endl;
```

Comment:
when a point load is applied using "assign_vector_by_direction_process" one gets the following error here:

`Trying to fix/free dof of variable POINT_LOAD_X but this dof does not exist in node #1!`

Reply:
Will be fixed/added in future PRs